spec/services/slack_notification_service_spec.rb

```diff
@@ -0,0 +1,48 @@
+require 'rails_helper'
+
+describe SlackNotificationService do
```

Comment:
```suggestion
describe SlackNotificationService do
    describe '#initialize' do
```

Reply:
Maybe. My idea was to explore the class code by creating scenarios. First, started with initialization scenarios, quickly finding issues with nil errors when Hash structure is not the expected one. Added a `raise` on #initialize and 🔁 but only one scenario did into the PR. I could add more scenarios or change it to `#initialize` as you proposed.

Reply:
Ok. I like this way